src/app/report-user/report-user.component.html

```diff
@@ -0,0 +1,13 @@
+<body>
+<div class="container" *ngIf="isLoggedIn">
+  <h3 style="text-align: left">Gebruiker rapporteren</h3>
+  <form role="form" (submit)="submitReport()">
+    <div class="form-group">
+      <textarea placeholder="Waarom wilt u deze gebruiker rapporteren?" class="form-control" id="reportMessage" rows="3" [(ngModel)]="report.message" name="message"></textarea>
+    </div>
+    <div class="d-flex justify-content-between align-items-center" style="margin-top: 45px">
+      <button id="submit-button" type="submit" class="btn btn-secondary" (click)="submitReport()">Rapporteer</button>
+    </div>
+  </form>
```

Comment:
Omdat er op de `form` en op de `button` allebei `submitReport()` wordt uitgevoerd, worden er 2 requests gestuurd ipv een. 

```suggestion
  <form role="form" (submit)="submitReport()">
    <div class="form-group">
      <textarea placeholder="Waarom wilt u deze gebruiker rapporteren?" class="form-control" id="reportMessage" rows="3" [(ngModel)]="report.message" name="message"></textarea>
    </div>
    <div class="d-flex justify-content-between align-items-center" style="margin-top: 45px">
      <button id="submit-button" type="submit" class="btn btn-secondary">Rapporteer</button>
    </div>
  </form>
```

Reply:
Ooh goed punt, heb de click over het hoofd gezien.